src/main/java/seedu/address/logic/commands/CommentCommand.java

```diff
@@ -19,19 +19,16 @@ public class CommentCommand extends Command {
 
     public static final String COMMAND_WORD = "comment";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Edits the comment of the person identified "
-            + "by the index number used in the last person listing. "
-            + "Existing comments will be overwritten by the input.\n"
-            + "Parameters: INDEX (must be a positive integer) "
-            + "c/ [COMMENT]\n"
-            + "Example: " + COMMAND_WORD + " 1 "
-            + "c/ Is always late to class.";
-
-    public static final String MESSAGE_NOT_IMPLEMENTED_YET =
-            "Comment command not implemented yet";
-
-    public static final String MESSAGE_ARGUMENTS = "Index: %1$d, Comment: %2$s";
+    public static final String MESSAGE_USAGE =
+            "Ensure the correct Format: " + COMMAND_WORD + " INDEX c/ COMMENT\n"
```

Comment:
The refactoring of MESSAGE_USAGE improves clarity but consider breaking it down into smaller constants for better readability.